site/src/components/gradient-card.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Gradient card
+description: The gradient card component in the Atlas Design System
+template: standard
+classType: Component
+classPrefixes:
+  - gradient-card
+---
+
+# Gradient card
+
+The `.gradient-card` class contains a predefined gradient border, border width and border radius for consistency. The inner `.gradient-card-content` element handles the border radius size and default background color for the card content and may be modified by Atomic classes.
```

Comment:
Please update the description to give a general explanation of what this component is and how/when can be used.

No need to demo its minor variations using atomics. If there are significant variations that can be created on top of this component using a set of different atomics it can be added to the "Patterns" section. But I have a feeling there is not a case here.

I wish I could give you a nicely composed sentence right away but that's definitely not my strongest side 😁

Reply:
@ollips thanks! I've updated the overview text and added usage.